sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+sonar.host.url=https://sonarcloud.io
+sonar.organization=sunbird-knowlg
+sonar.projectKey=sunbird-knowlg_knowlg-portal
+sonar.projectName=sunbird-knowledge
+sonar.projectVersion=1.0
+sonar.sources=src
+sonar.language=ts
+sonar.exclusions=**/*.module.ts, src/assets/**,server/**,src/app/**/*.spec.data.ts,src/app/**/*.spec.ts,**/node_modules/**
+sonar.typescript.lcov.reportPaths=coverage/sunbird-knowledge/lcov.info
```

Comment:
Same as new projectname if it is possible to use new name

Reply:
@vinukumar-vs Here we can not change to new name because if we change here than we need to do the same changes in    all the places